### ISSUE TITLE
Support Node Engine 20

### ISSUE THREAD

**Issue by daoleno** [lens-protocol/lens-sdk#789]
**Is your feature request related to a problem? Please describe.**

In my Node 20 project, I want to install Lens SDK, but I got an error.
```
error @lens-protocol/client@2.0.0-alpha.26: The engine "node" is incompatible with this module. Expected version "^18.15.0". Got "20.9.0"
```

**Describe the solution you'd like**

Allow node 20 version in engines config
https://github.com/lens-protocol/lens-sdk/blob/d3a85d1dde3f214da2ad8f29c9126f384ab9b8a6/packages/client/package.json#L99

**Comment by cesarenaldi** [comment]
The `@lens-protocol/client` does not have any hard dependency on Node 18. It just that when the project started Node 20 was not the stable release yet. Happy to relax this.

Given the error it seems your local setup has `engine-strict` checking set:

> Unless the user has set the [engine-strict config](https://docs.npmjs.com/cli/v10/using-npm/config#engine-strict) flag, this field is advisory only and will only produce warnings when your package is installed as a dependency.

To unblock you in the short term you can relax such constraint and the error will become an advisory warning.

It would be lovely if you could help us by reporting any issue in using it with Node 20.

**Comment by cesarenaldi** [comment]
Change for this will be part of next SDK release.

**Comment by cesarenaldi** [comment]
@daoleno this is now released: `@lens-protocol/client@2.0.0-alpha.28`. Please let me know it any issue.
